schemas/miscellaneous/colorMap.schema.tpl.json

```diff
@@ -1,24 +1,24 @@
 {
-    "_type": "https://openminds.ebrains.eu/sands/ColorMap",
-    "required": [
-        "name",
-        "codeSource",
-        "programmingLanguage"
-    ],
-    "properties": {
-      "codeSource": {
-        "type": "string",
-        "_instruction": "Enter the code to retrieve the color map values (e.g., 'import matplotlib as mpl; cmap = mpl.get_cmap(name)')."
-      },
-      "name": {
-        "type": "string",
-        "_instruction": "Enter name of the color map."
-      },
-      "programmingLanguage": {
-        "_instruction": "Add the programming language which the color map is defined in.",
-        "_linkedTypes": [
+  "_type": "https://openminds.ebrains.eu/sands/ColorMap",
+  "required": [
+    "codeSource",
+    "name",
+    "programmingLanguage"
+  ],
+  "properties": {
+    "codeSource": {
+      "type": "string",
+      "_instruction": "Enter the code to retrieve the color map values (e.g., 'import matplotlib as mpl; cmap = mpl.get_cmap(name)')."
+    },
+    "name": {
+      "type": "string",
+      "_instruction": "Enter a descriptive name for this color map."
+    },
+    "programmingLanguage": {
+      "_instruction": "Add the programming language in which the color map is defined in.",
```

Comment:
@apdavison "... in which the color map is defined in."  OR "... in which the color map is defined." ??